testing/src/test/scala/org/http4s/testing/ErrorReporting.scala

```diff
@@ -35,28 +27,12 @@ object ErrorReporting {
       val originalOut = System.out
       val originalErr = System.err
 
-      // Create blank permissions
-      val perm = new Permissions
-
-      // CodeSource domain for which these permissions apply (all files)
-      val certs: Array[java.security.cert.Certificate] = null
-      val code: CodeSource = new CodeSource(new URL("file:/*"), certs)
-
-      val domain = new ProtectionDomain(code, perm)
-      val domains = new Array[ProtectionDomain](1)
-      domains(0) = domain
-      val context = new AccessControlContext(domains)
-
       // Redirect output to dummy stream
       val fakeOutStream = new PrintStream(NullOutStream)
       val fakeErrStream = new PrintStream(NullOutStream)
       System.setOut(fakeOutStream)
       System.setErr(fakeErrStream)
-      try java.security.AccessController.doPrivileged(
```

Comment:
I don't recall why this was ever necessary.  Tests pass without it, and it's only used in tests.